atlas-db-controller/controller.go

```diff
@@ -244,6 +272,81 @@ func (c *Controller) processNextWorkItem(q workqueue.RateLimitingInterface, sync
 	return true
 }
 
+func (c *Controller) syncSchema(key string) error {
+	glog.Infof("Schema key: %v", key)
+
+	// Convert the namespace/name string into a distinct namespace and name
+	namespace, name, err := cache.SplitMetaNamespaceKey(key)
+	if err != nil {
+		runtime.HandleError(fmt.Errorf("invalid resource key: %s", key))
+		return err
+	}
+
+	// Get the Schema resource with this namespace/name
+	schema, err := c.schemasLister.DatabaseSchemas(namespace).Get(name)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			runtime.HandleError(fmt.Errorf("schema '%s' in work queue no longer exists", key))
+			return nil
+		}
+		return err
+	}
+	glog.Info(schema.Spec)
+	dbName := schema.Spec.Database
+	db, err := c.dbsLister.Databases(namespace).Get(dbName)
+	if err != nil {
+		runtime.HandleError(fmt.Errorf("cannot get database `%s`: %s", dbName, err))
+		return err
+	}
+
+	glog.Info(db.Spec.ServerType)
+	glog.Info(db.Spec.Dsn)
+
+	if db.Spec.ServerType != "postgres" { //  && db.Spec.ServerType != ...
```

Comment:
I'd like to support `mysql` too. It's good to have more than one to make sure we don't have any engine-specific stuff.